scripts/device-test.ps1

```diff
@@ -76,7 +76,7 @@ try
         if (!(Get-Command xharness -ErrorAction SilentlyContinue))
         {
             Push-Location ($CI ? $env:RUNNER_TEMP : $IsWindows ? $env:TMP : $IsMacos ? $env:TMPDIR : '/temp')
-            dotnet tool install Microsoft.DotNet.XHarness.CLI --global --version '10.0.0-prerelease*' `
+            dotnet tool install Microsoft.DotNet.XHarness.CLI --global --version '10.0.0-prerelease.25330.2' `
```

Comment:
note:
- XHarness repository: https://github.com/dotnet/xharness
- XHarness preview feed: https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-eng&view=versions&package=Microsoft.DotNet.XHarness.CLI&protocolType=NuGet

The error occurred in `10.0.0-prerelease.25360.1`.
Pinning to `10.0.0-prerelease.25330.2` fixed the issue.

Thanks @vaind